CMakeLists.txt

```diff
@@ -44,17 +44,20 @@ macro(build_uncrustify)
     set(patch_exe patch)
   endif()
 
+  # Pinning to tip of master at the time of a desired bugfix
+  set(uncrustify_version 45b836cff040594994d364ad6fd3f04adc890a26)
+  set(uncrustify_version_hash 7cc32ad800c8d639bdf145e2a113a66b)
   # Get uncrustify 0.68.1
```

Comment:
Keeping this comment unchanged is pretty confusing since the code doesn't use that specific version anymore.